src/demo/ui.jsx

```diff
@@ -603,7 +603,7 @@ const styles = {
   },
   arrowUpperRight: {
     top: '13%',
-    right: '-3.5%',
+    right: '-2%',
```

Comment:
Actually do we know where the arrow should be shown, since the info button is never guaranteed?

Reply:
Oh, I think we only use arrow `UpperRight` on this early level so maybe nothing to worry about.